airbyte-cdk/python/airbyte_cdk/sources/streams/concurrent/cursor.py

```diff
@@ -26,12 +26,12 @@ def __lt__(self: "Comparable", other: "Comparable") -> bool:
 
 class CursorField:
     def __init__(self, cursor_field_key: str) -> None:
-        self._cursor_field_key = cursor_field_key
+        self.cursor_field_key = cursor_field_key
```

Comment:
curious: why does `cursor_field_key` need to be public?

Reply:
I'm using it [here](https://github.com/airbytehq/airbyte/pull/33531/files#diff-44999ccc78e8a64a5a79f8352f8aa2a45d8dde6ecaa92d3dff0cc773de104197R90) and [here](https://github.com/airbytehq/airbyte/pull/33531/files#diff-44999ccc78e8a64a5a79f8352f8aa2a45d8dde6ecaa92d3dff0cc773de104197R119).